import-service/handlers/importFileParser/csvParser.test.js

```diff
@@ -0,0 +1,44 @@
+import csvParser from 'csv-parser';
+import fs from 'fs';
+import path from 'path';
+
+const fixture = path.resolve('__fixtures__/import-test.csv');
+
+it('parses CSV file', async () => {
```

Comment:
I think you should test your code. This test checks `csv-parser` npm package functionality

Reply:
Yes and no, Alex :) Tests for help, a fixture that I have is a part of code BTW, but that test helped me to find hidden symbols when I exported my CSV from Excel.